packages/app-desktop/gui/NoteSearchBar.jsx

```diff
@@ -120,8 +121,10 @@ class NoteSearchBarComponent extends React.Component {
 		if (!this.props.searching) {
 			if (this.props.resultCount === 0 && query.length > 0) {
 				this.backgroundColor = theme.warningBackgroundColor;
+				this.isSearchResultFound = false;
 			} else {
 				this.backgroundColor = theme.backgroundColor;
+				this.isSearchResultFound = true;
```

Comment:
There's no reason for this to be a class attribute. please change it to a local variable.